app/src/main/java/org/simple/clinic/AppDatabase.kt

```diff
@@ -51,4 +56,25 @@ abstract class AppDatabase : RoomDatabase() {
   abstract fun prescriptionDao(): PrescribedDrug.RoomDao
 
   abstract fun facilityDao(): Facility.RoomDao
+
+  abstract fun userDao(): LoggedInUser.RoomDao
+
+  class Migration_3_4 : Migration(3, 4) {
+
+    override fun migrate(database: SupportSQLiteDatabase) {
+      database.execSQL("""
+        CREATE TABLE IF NOT EXISTS `LoggedInUser` (
```

Comment:
Can we add an android test to run the migrations?

Reply:
Let's skip this. I have to make database changes myself and I'll implement this.